README.md

```diff
@@ -1,68 +1,188 @@
 
-**❗ In order to start using Flutter Deep Face Cam you must have the [Flutter SDK][flutter_install_link] installed on your machine.**
+### Key Features:
+- **Real-Time Image Processing** 
+- **Image-to-Image Processing** 
+- **Video-to-Image Processing** 
 
-Install via `flutter pub add`:
+---
 
-```sh
-dart pub add flutter_deep_face_cam
-```
+## Getting Started
 
----
+Before you start using the package, ensure you have a Flutter environment set up. You can find instructions on how to set up Flutter on the [official Flutter website](https://flutter.dev/docs/get-started/install).
+
+### Installation
+
+1. **Add Dependency**  
+   Add `deep_live_cam` as a dependency in your `pubspec.yaml` file:
+
+   ```yaml
+   dependencies:
+     deep_live_cam: ^1.0.0
+   ```
 
-## Continuous Integration 🤖
+2. **Install the Package**  
+   Run the following command in your project directory:
 
-Flutter Deep Face Cam comes with a built-in [GitHub Actions workflow][github_actions_link] powered by [Very Good Workflows][very_good_workflows_link] but you can also add your preferred CI/CD solution.
+   ```bash
+   flutter pub get
+   ```
 
-Out of the box, on each pull request and push, the CI `formats`, `lints`, and `tests` the code. This ensures the code remains consistent and behaves correctly as you add functionality or make changes. The project uses [Very Good Analysis][very_good_analysis_link] for a strict set of analysis options used by our team. Code coverage is enforced using the [Very Good Workflows][very_good_coverage_link].
+3. **Import the Package**
+
+   Import the package in your Dart files:
+
+   ```dart
+   import 'package:deep_live_cam/deep_live_cam.dart';
+   ```
 
 ---
 
-## Running Tests 🧪
+## Usage
+
+Now that you've installed the package, let’s walk through how to use it in your Flutter application.
+
+Before you can start using the Deep Live Cam Flutter package, you'll need to follow these steps:
+1. Create an Application on flutterdeeplivecam.com
+
+    Go to the flutterdeeplivecam.com website.
+    Sign up or log in to your account.
+    Navigate to the Dashboard.
+    Create a new application by providing a name and description for your app.
+    Once your app is created, you'll be provided with an API Key.
+
+This API key will be used to authenticate your requests and access the image and video processing services.
 
-For first time users, install the [very_good_cli][very_good_cli_link]:
+2. Add the API Key to Your Flutter Application
 
-```sh
-dart pub global activate very_good_cli
+Once you have your API key, you need to add it to your Flutter project. Here’s how:
+
+    Open your pubspec.yaml file and add the API key under the Deep Live Cam configuration section:
+
+dependencies:
+  deep_live_cam: ^1.0.0
+
+flutter:
+  deep_live_cam:
+    api_key: "YOUR_API_KEY"
+
+Replace "YOUR_API_KEY" with the API key you generated on the flutterdeeplivecam.com website.
+
```

Comment:
We can't have the security key in the pubspec.yaml file. It would be a huge security issue. I also shouldn't have to be checked to git. 

The best case scenario would be to have a custom remote config implementation. 

But for now, we can think of an env variable. 

@yunweneric @nicowalter256 @AsnelChristian

Reply:
We will also start with ver: 0.0.1, not 1.0.0

Reply:
Have you thought of using dart defined env variables?
I could help you with this